docs/src/starknet/verify.md

```diff
@@ -26,7 +30,7 @@ Then run:
 ```shell
 $ sncast \
     verify \
-    --contract-address 0x01e4ebe3278ab4633a9d0d3f5c4290001f29bc3179a70e570b6817dd7f8264fa \
+    --contract-address 0x8448a68b5ea1affc45e3fd4b8b480ea36a51dc34e337a16d2567d32d0c6f8b \
```

Comment:
Why was this address changed? It doesn't seem like a class hash of any existing contract